### PR TITLE
[RFC] Windows: fix serverstart functional test

### PR DESCRIPTION
**Pull request by sethjackson**
Cherry picked from #810. From equalsraf@a514dff.

### CODE REVIEW

File: test/functional/helpers.lua [hunk 1]
```diff
@@ -438,4 +439,5 @@ return {
   curbufmeths = curbufmeths,
   curwinmeths = curwinmeths,
   curtabmeths = curtabmeths,
+  ffi = ffi,
```
Comment (justinmk) [inline test/functional/helpers.lua]: This doesn't need to be served by the helpers module; test suites should just `require('ffi')` themselves.

Comment (sethjackson) [inline test/functional/helpers.lua]: Ok. There will be more suites that require `ffi`.

Comment (justinmk) [inline test/functional/helpers.lua]: If it's only for the OS check then we can eventually add a helper function like `get_os()`. I never liked the idea of depending on FFI for this--we could just use `nvim` for the purpose instead, e.g. `helpers.eval('has("win32")')`.

Comment (sethjackson) [inline test/functional/helpers.lua]: Yea. AFAIK we are only using FFI for the OS check here and in another pending test fix. So I can change this to the above if you want.

Comment (justinmk) [inline test/functional/helpers.lua]: If you don't mind

Comment (sethjackson) [inline test/functional/helpers.lua]: Sure thing. Done.

**Pull request merged**